### PR TITLE
fix: landing page styling of widget

### PR DESCRIPTION
**Pull request by just-toby**
fixes the landing page styling for the widget-enabled variant (see preview link for the difference between test and control)

### CODE REVIEW

File: src/graphql/thegraph/__generated__/types-and-hooks.ts [hunk 1]
```diff
@@ -37,6 +37,7 @@ export type Bundle = {
 export type Bundle_Filter = {
```
Comment (zzmp) [inline src/graphql/thegraph/__generated__/types-and-hooks.ts:37]: This should be changed in a separate PR, right?

File: src/pages/Landing/index.tsx [hunk 2]
```diff
@@ -306,9 +326,15 @@ export default function Landing() {
               name={SharedEventName.ELEMENT_CLICKED}
               element={InterfaceElementName.LANDING_PAGE_SWAP_ELEMENT}
             >
-              <Link to="/swap">
-                <LandingSwap />
-              </Link>
+              {swapWidgetEnabled ? (
```
Comment (zzmp) [inline src/pages/Landing/index.tsx:329]: nit: You can inline this: `{useSwapWidgetEnabled() ? (`.

**Review by zzmp — CHANGES_REQUESTED** [review]

File: src/pages/Landing/index.tsx (hunk 2 above)
Comment (just-toby) [reply src/pages/Landing/index.tsx:329]: i tried that but we can't because it would be conditionally called

File: src/graphql/thegraph/__generated__/types-and-hooks.ts (hunk 1 above)
Comment (just-toby) [reply src/graphql/thegraph/__generated__/types-and-hooks.ts:37]: i think we're supposed to check these changes in, according to @cbachmeier

Comment (zzmp) [reply src/graphql/thegraph/__generated__/types-and-hooks.ts:37]: but nothing in this PR should have generated this. It's not related to this PR's changes, and should be checked in through a separate PR.

**Review by zzmp — CHANGES_REQUESTED** [review]
types-and-hooks.ts should still be reverted

File: src/graphql/thegraph/__generated__/types-and-hooks.ts (hunk 1 above)
Comment (just-toby) [reply src/graphql/thegraph/__generated__/types-and-hooks.ts]: ok, removed

**Review by zzmp — APPROVED** [review]
